neo/app/views/cloudberry/index.scala.html

```diff
@@ -0,0 +1,32 @@
+@import play.api.libs.json.JsValue
+@import play.api.libs.json.Json
+@(datasets: Seq[JsValue], form: Form[Cloudberry.RegisterForm])(implicit messages: play.api.i18n.Messages)
+
+@import views.html.cloudberry.textarea
+
+@main("Cloudberry Admin") {
+    <h3>Registered Datasets</h3>
+    <div class="list-group">
+    @for(d <- datasets) {
+        <li class="list-group-item">@{
+            Json.prettyPrint(d)
```

Comment:
You can wrap the json using the `<pre>` tag to print the formatted Json (assuming this Json.prettyPrint(d) has already prettified the Json). Find out more here: https://www.w3schools.com/tags/tag_pre.asp. 

Also, check out Google's `code-prettify` plugin to make it even prettier (like syntax highlighting...). You can do so by adding `<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>`, and then add the attribute `class="prettyprint"` to the `pre` tag. Find out more in here: https://github.com/google/code-prettify